--- src/AddEditDeleteUpdateIndexViewAdapter.php
+++ PHP_CodeSniffer
@@ -6,7 +6,8 @@
 
     public $tableName;
     
-    public function __construct($connection, $tableName){
+    public function __construct($connection, $tableName)
+    {
         parent::__construct($connection);
         $this->tableName = $tableName;
     }
@@ -16,7 +17,8 @@
         parent::update($id, $this->tableName, $data);
     }
 
-    public function index(){
+    public function index()
+    {
         $query = $this->basicDB->select($this->tableName)
         ->run();
         
@@ -37,7 +39,8 @@
         parent::create($data, $this->tableName);
     }
 
-    public function delete($id){
+    public function delete($id)
+    {
         parent::delete($id, $this->tableName);
     }
-}
\ No newline at end of file
+}
--- src/AddEditDeleteUpdateIndexViewInterface.php
+++ PHP_CodeSniffer
@@ -13,4 +13,4 @@
     public function view($id);
 
     public function delete($id);
-}
\ No newline at end of file
+}
--- src/BasicDB.php
+++ PHP_CodeSniffer
@@ -1,9 +1,10 @@
 <?php
 
 namespace Erbilen\Database;
+
 /**
  * Class BasicDB
- * 
+ *
  * @author Tayfun Erbilen
  *         @web http://www.erbilen.net
  *         @mail tayfunerbilen@gmail.com
@@ -123,7 +124,7 @@
      *            $username
      * @param
      *            $password
-     * @param string $charset            
+     * @param string $charset
      */
     public function __construct($host, $dbname, $username, $password, $charset = 'utf8')
     {
@@ -166,8 +167,8 @@
      *            $column
      * @param
      *            $value
-     * @param string $mark            
-     * @param bool $filter            
+     * @param string $mark
+     * @param bool $filter
      * @return $this
      */
     public function where($column, $value = '', $mark = '=', $logical = '&&')
@@ -205,7 +206,7 @@
      *            $targetTable
      * @param
      *            $joinSql
-     * @param string $joinType            
+     * @param string $joinType
      * @return $this
      */
     public function join($targetTable, $joinSql, $joinType = 'inner')
@@ -219,7 +220,7 @@
      *
      * @param
      *            $columnName
-     * @param string $sort            
+     * @param string $sort
      */
     public function orderby($columnName, $sort = 'ASC')
     {
@@ -258,7 +259,7 @@
     /**
      * Used for running Insert/Update/Select operations.
      *
-     * @param bool $single            
+     * @param bool $single
      * @return array|mixed
      */
     public function run($single = false)
@@ -284,10 +285,9 @@
         
         $query = $this->query($this->sql);
         
-        if ($single){
+        if ($single) {
             return $query->fetch(parent::FETCH_ASSOC);
-        }
-        else{
+        } else {
             return $query->fetchAll(parent::FETCH_ASSOC);
         }
             
--- src/CrudableAdapter.php
+++ PHP_CodeSniffer
@@ -5,11 +5,13 @@
 {
     private $basicDB;
     
-    public function __construct(BasicDB $connection){
+    public function __construct(BasicDB $connection)
+    {
         $this->basicDB = $connection;
     }
 
-    public function create($data,$tableName){
+    public function create($data, $tableName)
+    {
         
         $query = $this->basicDB->insert($tableName)->set($data);
         
@@ -17,7 +19,8 @@
         
     }
 
-    public function read($id, $tableName){
+    public function read($id, $tableName)
+    {
         $query = $this->basicDB->select($tableName)
         ->where('id', $id)
         ->run();
@@ -30,7 +33,8 @@
         
     }
 
-    public function update($id, $tableName ,$data){
+    public function update($id, $tableName, $data)
+    {
         // update
         $query = $this->basicDB->update($tableName)
         ->where('id', $id)
@@ -41,7 +45,8 @@
         }
     }
 
-    public function delete($id,$tableName){
+    public function delete($id, $tableName)
+    {
         // delete
         $query = $this->basicDB->delete($tableName)
         ->where('id', $id)
@@ -51,4 +56,4 @@
             return true;
         }
     }
-}
\ No newline at end of file
+}
--- src/CrudableInterface.php
+++ PHP_CodeSniffer
@@ -11,4 +11,4 @@
     public function update($id, $data);
 
     public function delete($id);
-}
\ No newline at end of file
+}
--- tests/BasicDBTest.php
+++ PHP_CodeSniffer
@@ -30,7 +30,8 @@
      * This method is called after a test is executed.
      */
     protected function tearDown()
-    {}
+    {
+    }
 
     protected function getDataSet()
     {
@@ -128,6 +129,4 @@
     
         $this->assertNotEmpty($query);
     }
-
- 
 }

